### ISSUE TITLE
null pointer used in bxgeomtools/src/rotated_boxed_model.cc

### ISSUE THREAD

**Issue by emchauve** [BxCppDev/Bayeux#74]
https://github.com/BxCppDev/Bayeux/blob/2686ef5a93f446ed9345213a0e15c4b2b3ba51a9/source/bxgeomtools/src/rotated_boxed_model.cc#L171-L175

The compiler is auto detecting the null pointer occurrence (`the_model->get_name() ` on line 174) !

**Comment by goliviero** [comment]
Same error as Manu when compiling Bayeux 3.5.3. 
It implies this compilation error : 
```
source/bxgeomtools/src/rotated_boxed_model.cc:174:65: error: ‘this’ pointer is null [-Werror=nonnull]
  174 |                    "The rotating model '" << the_model->get_name() << "' is not stackable in model '" << get_name() << "'!");
      |                                              ~~~~~~~~~~~~~~~~~~~^~

**Comment by fmauger** [comment]
This bug has been fixed  in my ubuntu 22.04 port branch. I have to commit it.

**Comment by fmauger** [comment]
The develop branch (pre 3.5.4) includes the fix. Please check !
